### DISCUSSION TITLE
Question about Dark Mode disabled on "older" versions of Windows 10

### DISCUSSION THREAD

**Discussion by Gameboi64** [reupen/columns_ui#825] [General] [answered]
I have to ask, why is it locked behind certain versions of Windows 10? I run 1809, and unfortunately I'm not able to update past that due to the version of Windows I use. However, I upgraded to Foobar2000 2.0 and it supports dark mode on my operating system seamlessly. Was this chosen because it couldn't be tested on versions below that, or was there some dependency reason?

**Comment by reupen** [comment, marked as answer]
It is the latter. Dark mode in Columns UI uses functionality that was implemented in Windows 10 version 2004.
